src/main/java/org/demo/lesson4/tasks/task3.java

```diff
@@ -0,0 +1,25 @@
+package org.demo.lesson4.tasks;
+
+import java.util.*;
+
+
+public class task3 {
```

Comment:
Better to name Task3 in accordance with the code conventions, example: https://www.geeksforgeeks.org/java-naming-conventions/ :)